dockerfiles/Dockerfile.env.dist

```diff
@@ -0,0 +1,3 @@
+POSTGRES_NAME=StreamWebs
```

Comment:
It's unusual, but not actually wrong, to capitalize usernames, is there a reason?